From 1f60d597b2a2d6b7ba78375a71dae8901132e222 Mon Sep 17 00:00:00 2001
From: Subrat Meher <smeher02.aoe3@gmail.com>
Date: Fri, 29 May 2020 20:02:48 +0200
Subject: [PATCH] Fix openzwave include

---
 CMakeLists.txt | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/CMakeLists.txt b/CMakeLists.txt
index 444b017f89cf62f1bf9b0bb6a2fa7af392004c3d..336c7a967c5c07bf3359159cf5a6311f680a282e 100644
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -404,7 +404,7 @@ ELSE(LUA_LIBRARIES AND LUA_INCLUDE_DIRS)
   IF(LUA_FOUND)
     MESSAGE(STATUS "LUA library found at  : ${LUA_LIBRARIES}")
     MESSAGE(STATUS "LUA includes found at : ${LUA_INCLUDE_DIR}")
-    INCLUDE_DIRECTORIES(${LUA_INCLUDE_DIR}) 
+    INCLUDE_DIRECTORIES(${LUA_INCLUDE_DIR})
   ELSE(LUA_FOUND)
     MESSAGE(FATAL_ERROR "LUA 5.3 not found! use sudo apt-get install liblua5.3-dev")
   ENDIF(LUA_FOUND)
@@ -703,7 +703,7 @@ IF(OpenZWave)
     IF(CMAKE_SYSTEM_NAME STREQUAL "FreeBSD")
       include_directories(${OPENZWAVE_INCLUDE_DIRS})
     ELSE()
-      include_directories(${OPENZWAVE_INCLUDE_DIRS}/openzwave)
+      include_directories(${OPENZWAVE_INCLUDE_DIRS})
     ENDIF()
     message(STATUS "OpenZWave includes found at: ${OPENZWAVE_INCLUDE_DIRS}")
   ELSE()
-- 
2.26.2

